examples/evm-to-btc-fungible-transfer/README.md

```diff
@@ -0,0 +1,81 @@
+# Sygma SDK EVM to BTC example
+
+This is an example script that demonstrates the functionality of the SDK using the Sygma ecosystem. The script showcases a token transfer between the provided account and a BTC testnet account. 
```

Comment:
This is an example that demonstrates functionality of the protocol using Sygma SDK. The `src/transfer.ts` script showcases how bitcoins can transferred from an EVM Sepolia address utilizing `@buildwithsygma/evm` package.